api/db.py

```diff
@@ -27,11 +27,16 @@ class Database:
     def __init__(self, host='db', db_name='kernelci'):
         self._motor = motor_asyncio.AsyncIOMotorClient(host=host)
         self._db = self._motor[db_name]
+        self._create_index()
 
     def _get_collection(self, model):
         col = self.COLLECTIONS[model]
         return self._db[col]
 
+    def _create_index(self):
+        col = self._get_collection(User)
+        col.create_index("username", unique=True)
+
```

Comment:
This information is related to the `User` model, which should be kept outside of the `Database` class.

As the `Database` class has a `COLLECTIONS` attribute with all the models and collections, it could iterate over it and call a `create_indexes` method on each model class which would then be able to define their own indexes.